doc/api/util.md

```diff
@@ -1237,6 +1237,10 @@ The encoding supported by the `TextEncoder` instance. Always set to `'utf-8'`.
 ## `util.types`
 <!-- YAML
 added: v10.0.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/34055
+    description: Exposed as `require('util/types')`
```

Comment:
Linter demands this. I tried pushing to your branch to expedite it but it wouldn't let me.

```suggestion
    description: Exposed as `require('util/types')`.
```